hips/tiles/tests/test_surveys.py

```diff
@@ -85,8 +96,6 @@ def test_fetch(self):
         surveys = HipsSurveyPropertiesList.fetch()
         assert len(surveys.data) > 3
 
-        # TODO: look up survey by name here
-        # Otherwise this will break when the list changes
-        survey = surveys.data[0]
+        survey = HipsSurveyProperties.from_name('CDS/P/2MASS/H')
```

Comment:
I've updated my comment above: the class names I gave were wrong. I meant adding a normal method `HipsSurveyPropertiesList.from_name` (not a classmethod), because really looking for a name is an operation on the list, i.e. the code that does the looping through surveys and looking for a name ID should be a method on `HipsSurveyPropertiesList`.

Is it clear now?